src/test/java/com/meilisearch/integration/TasksTest.java

```diff
@@ -0,0 +1,92 @@
+package com.meilisearch.integration;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.meilisearch.integration.classes.AbstractIT;
+import com.meilisearch.integration.classes.TestData;
+import com.meilisearch.sdk.Index;
+import com.meilisearch.sdk.Task;
+import com.meilisearch.sdk.utils.Movie;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Tag("integration")
+public class TasksTest extends AbstractIT {
+
+    private TestData<Movie> testData;
+
+    @BeforeEach
+    public void initialize() {
+        this.setUp();
+        if (testData == null) testData = this.getTestData(MOVIES_INDEX, Movie.class);
+    }
+
+    @AfterAll
+    static void cleanMeiliSearch() {
+        cleanup();
+    }
+
+    /** Test Get Task */
+    @Test
+    public void testClientGetTask() throws Exception {
+        String indexUid = "GetClientTask";
+        Task response = client.createIndex(indexUid);
+        client.waitForTask(response.getUid());
+
+        Task task = client.getTask(response.getUid());
+        assertTrue(task instanceof Task);
+        assertNotNull(task.getStatus());
+        assertNotEquals("", task.getStatus());
+        assertNotNull(task.getStartedAt());
+        assertNotNull(task.getEnqueuedAt());
+        assertNotNull(task.getFinishedAt());
+        assertTrue(task.getUid() >= 0);
```

Comment:
What do you think about using something like this: https://code.google.com/archive/p/hamcrest/wikis/Tutorial.wiki

In order to enable us a better reading of the code, and the errors in tests (eg. https://stackoverflow.com/a/18766311)

Reply:
If you plan to add a testing library I recommend looking at `AssertJ`, it is much more pleasant to work with since it has a fluent API which the IDE can autocomplete for you (eg `assertThat(task.getUid()).isGreaterThanOrEqualTo(0)`)

(but as a user waiting for the Java API to catch up with the latest meilisearch, I would say polish the tests after getting this merged :smile: )

Reply:
haha yeaah you're right @uwemaurer I'll do these kinds of things in the next iteration, not here mixing the release and the improvements :D 

But I liked your suggestion, actually in the reviews I usually like to raise the discussion 🔥

Reply:
I will create issue #324  to add a testing library and redo them all right !